### PR TITLE
Added explanation when testing hybrid with a backend running on your …

### PR DESCRIPTION
**Pull request by letzya**
…localhost

### CODE REVIEW

**Review by marksou — APPROVED** [review]
Made some minor edits.

**Comment by marksou** [comment]
@letzya I believe from @buger that this page needs some work still?

File: tyk-docs/content/get-started/with-tyk-hybrid/tutorials/create-api.mmark [hunk 1]
```diff
@@ -39,3 +39,40 @@ Output:
...
+        -p 80:80 \
+        -e PORT=$PORT \
+        -e SECRET=$SECRET \
+        -e ORGID=$ORGID \
+        -e APIKEY=$APIKEY \
+        -e REDISHOST=$REDISHOST \
+        -e REDISPW=$REDISPW \
+        -e RPORT=$RPORT \
+        -e BINDSLUG=1 \
+        --net=host \
```
Comment (letzya) [inline tyk-docs/content/get-started/with-tyk-hybrid/tutorials/create-api.mmark]: Still need to work on this doc.
```suggestion
        --net=host \
```

**Comment by buger** [comment]
@marksou needs fixing conflict

**Comment by marksou** [comment]
@buger fixed and merged.

**Pull request merged**